snippets/ztp-storage-lvms.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: SNIPPET
+.Recommended LVMCluster configuration
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: odf-lvmcluster
+  namespace: openshift-storage
+spec:
+  storage:
+    deviceClasses:
+    - name: vg1
+      thinPoolConfig:
+        name: thin-pool-1 <1>
+        overprovisionRatio: 10
+        sizePercent: 90
+----
+<1> If no disks are specified in the `deviceSelector.paths` field, the {lvms} uses all the unused disks in the specified thin pool.
```

Comment:
Where is the `deviceSelector.paths` field?

Reply:
Added an example path for clarification, so the callout makes sense now